src/ds/d500/d500-private.cpp

```diff
@@ -43,18 +43,32 @@ namespace librealsense
         rs2_intrinsics get_d500_intrinsic_by_resolution(const vector<uint8_t>& raw_data, d500_calibration_table_id table_id, 
             uint32_t width, uint32_t height, bool is_symmetrization_enabled)
         {
-            switch (table_id)
-            {
-            case d500_calibration_table_id::depth_calibration_id:
+            
+            if (!raw_data.empty()) 
             {
-                return get_d500_depth_intrinsic_by_resolution(raw_data, width, height, is_symmetrization_enabled);
+                switch (table_id)
+                {
+                case d500_calibration_table_id::depth_calibration_id:
+                {
+                    return get_d500_depth_intrinsic_by_resolution(raw_data, width, height, is_symmetrization_enabled);
+                }
+                case d500_calibration_table_id::rgb_calibration_id:
+                {
+                    return get_d500_color_intrinsic_by_resolution(raw_data, width, height);
+                }
+                default:
+                    throw invalid_value_exception(rsutils::string::from() << "Parsing Calibration table type " << static_cast<int>(table_id) << " is not supported");
+                }
             }
-            case d500_calibration_table_id::rgb_calibration_id:
+            else // In case we got an empty table we will run with default values
             {
-                return get_d500_color_intrinsic_by_resolution(raw_data, width, height);
-            }
-            default:
-                throw invalid_value_exception(rsutils::string::from() << "Parsing Calibration table type " << static_cast<int>(table_id) << " is not supported");
+                LOG_ERROR("cannot read intrinsic values, setting defaults");
```

Comment:
please replace "setting defaults" with "using default values" - so that this log looks the same as the other one above

Reply:
Done